.golangci.yml

```diff
@@ -0,0 +1,18 @@
+linters:
```

Comment:
Perhaps we should enable this one here as well; https://github.com/containerd/containerd/pull/5180

Reply:
done